brig/cmd/brig/commands/project_create.go

```diff
@@ -0,0 +1,540 @@
+		}
+	}
+
+	if len(p.Secrets) > 0 {
+		fmt.Println("The following secrets are already defined:")
+		for k := range p.Secrets {
+			fmt.Printf("\t- %s\n", k)
+		}
+		fmt.Println("  (To remove a secret, enter the key for the name, and '-' as the value)")
+	}
+
+	addSecrets := false
+	if err := survey.AskOne(&survey.Confirm{
+		Message: "Add secrets?",
+		Help:    "Add key/value pairs for things like account names and tokens. These are available to your brigade.js",
+	}, &addSecrets, nil); err != nil {
+		return fmt.Errorf(abort, err)
+	} else if addSecrets {
+		kvp := struct {
+			Another bool
+			Key     string
+			Value   string
+		}{
+			Another: true,
+		}
+		for i := 1; kvp.Another; i++ {
+			err := survey.Ask([]*survey.Question{
+				{
+					Name:   "key",
+					Prompt: &survey.Input{Message: fmt.Sprintf("\tSecret %d", i)},
+				},
+				{
+					Name:   "value",
+					Prompt: &survey.Input{Message: "\tValue"},
+				},
+				{
+					Name:   "another",
+					Prompt: &survey.Confirm{Message: "===> Add another?"},
+				},
+			}, &kvp)
+			if err != nil {
+				return fmt.Errorf(abort, err)
+			}
+			if kvp.Value == "-" {
+				delete(p.Secrets, kvp.Key)
+				continue
+			}
+			p.Secrets[kvp.Key] = kvp.Value
+		}
+
+	}
+
+	if p.SharedSecret != "" {
+		fmt.Println("Auto-generating a Shared Secret...")
+		p.SharedSecret, _ = goutils.RandomAlphaNumeric(24)
+	}
+
+	configureGitHub := false
+	if err := survey.AskOne(&survey.Confirm{
+		Message: "Configure GitHub Enterprise or OAuth?",
+		Help:    "Configure GitHub CI/CD integration for this project",
+	}, &configureGitHub, nil); err != nil {
+		return fmt.Errorf(abort, err)
+	} else if configureGitHub {
+		if err := survey.Ask([]*survey.Question{
+			{
+				Name: "token",
+				Prompt: &survey.Input{
+					Message: "OAuth2 Token",
+					Help:    "Used for contacting the GitHub API. GitHub issues this.",
+					Default: p.Github.Token,
+				},
+			},
+			{
+				Name: "baseURL",
+				Prompt: &survey.Input{
+					Message: "GitHub Enterprise URL",
+					Help:    "If using GitHub Enterprise, set the base URL here",
+					Default: p.Github.BaseURL,
+				},
+			},
+			{
+				Name: "uploadURL",
+				Prompt: &survey.Input{
+					Message: "GitHub Enterprise Upload URL",
+					Help:    "If using GitHub Enterprise, set the upload URL here",
+					Default: p.Github.UploadURL,
+				},
+			},
+		}, &p.Github); err != nil {
+			return fmt.Errorf(abort, err)
+		}
+	}
+
+	configureK8s := false
+	if err := survey.AskOne(&survey.Confirm{
+		Message: "Configure advanced Kubernetes options?",
+		Help:    "Set advanced options for custom VCS images, storage, and permissions",
+	}, &configureK8s, nil); err != nil {
+		return fmt.Errorf(abort, err)
+	} else if configureK8s {
+		if err := survey.Ask([]*survey.Question{
+			{
+				Name: "namespace",
+				Prompt: &survey.Input{
+					Message: "Kubernetes Namespace",
+					Help:    "The namespace in which this project applies",
+					Default: p.Kubernetes.Namespace,
```

Comment:
This is not used, see comment above.

Reply:
Oh, you're right... the value here should just be set to `globalNamespace`.